ocs_ci/ocs/cluster.py

```diff
@@ -912,6 +912,74 @@ def get_osd_utilization():
     return osd_filled
 
 
+def get_ceph_df_detail():
+    """
+    Get ceph osd df detail
+    :return: dict returned from cmd
+    """
+    ceph_cmd="ceph df detail"
+    ct_pod = pod.get_ceph_tools_pod()
+    output = ct_pod.exec_ceph_cmd(ceph_cmd=ceph_cmd)
+    return output
+
+
+def validate_replica_data(pool_name, replica):
+    """
+    Check if data is replica 2 or 3
+    :param int replica: size of the replica(2,3)
+    :param string pool_name: name of the pool to check replica
+    :return: true if replica is verified, false if not
+    """
+    from ocs_ci.ocs.exceptions import PoolNotFound
```

Comment:
Move this import in the beginning of the python file, among other imports.

Reply:
moved the import to the start of the file.

Reply:
I still see the import there.

Reply:
same

Reply:
same :)